simularium/simularium-website
# store current color selections in local browser storage

Time estimate or Size
=======
_x-small_

Problem
=======
Closes #580, advances #511

Solution
========
Store user color selections as `UIDisplayData` in local storage in the browser.

Behavior is limited in current scope, no need to store defaults, and no application of any stored settings.

Whenever a user applies a color change, store a snapshot of the current color selections in local storage, using simularium file name as key.

Steps to Verify:
----------------
1. Open dev tools Application tab, storage.
2. Make a color change, key: value of current trajectory name and current color selections should be in local storage.
